library/firewall_lib.py

```diff
@@ -173,6 +184,48 @@
 except ImportError:
     HAS_FIREWALLD = False
 
+CONFIGS_TO_DELETE = [
+    "/etc/firewalld/zones/*",
+    "/etc/firewalld/policies/*",
+    "/etc/firewalld/direct.xml",
+]
+
+
+def hash_file(file_name):
+    out = check_output(["xmllint", "--c14n", file_name])
+    # return hashlib.md5(out).hexdigest()
+    return file_name
+
+
+def get_hashes_of_all_configs():
+    hashes = set()
+    for config in CONFIGS_TO_DELETE:
+        files = glob.glob(config)
+        for config_file in files:
+            if config_file.endswith(".old"):
```

Comment:
Rather than excluding files - instead include only files with a certain extension - that is, is it safe to say that only files ending in `.xml` should be considered?